src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -1332,14 +1334,14 @@ public void LanguageVersionAdded_Canary()
             // - update the "UpgradeProject" codefixer
```

Comment:
Did you update the UpgradeProject codefixer? We should add a diagnostic for "error CS80XX: Feature is not available in C# 7." and handle it in `CSharpUpgradeProjectCodeFixProvider`.
This also mean updating the method for checking feature availability.